torch/csrc/distributed/rpc/init.cpp

```diff
@@ -123,6 +123,11 @@ PyObject* rpc_init(PyObject* /* unused */) {
     agent->start();
   });
 
+  module.def("local_shutdown_rpc", []() {
```

Comment:
Lets call this `shutdown` and since this is a public API, lets add appropriate docs explaining what this function does.